src/iamclient/iamclient.cpp

```diff
@@ -270,8 +270,12 @@ void IAMClient::HandleChannels()
 
         if (auto err = SetupChannel(); !err.IsNone()) {
             LOG_ERR() << "Can't setup channel: err=" << err;
+            LOG_DBG() << "Reconnect in " << cReconnectInterval / 1000000 << " ms";
```

Comment:
is ms -> milli or microseconds?

Reply:
it is the standard symbol for milliseconds. Microseconds have symbol us.